src/renderer/shared/core/types/chain.ts

```diff
@@ -15,7 +15,7 @@ export type Chain = {
   options?: ChainOptions[];
 };
 
-export type ChainOptions = 'testnet' | 'crowdloans' | 'multisig' | 'regular_proxy' | 'ethereum_based';
+export type ChainOptions = 'testnet' | 'crowdloans' | 'multisig' | 'regular_proxy' | 'pure_proxy' | 'ethereum_based';
```

Comment:
This `type` has gotten so many options, nice to have enum instead of endless options of strings 😎
Could you create an issue and link it here with TODO?